it/uk/gov/hmrc/fhregistrationfrontend/controllers/ApplicationControllerIntegrationSpec.scala

```diff
@@ -25,126 +15,82 @@ class ApplicationControllerIntegrationSpec
 
       WsTestClient.withClient { client ⇒
         whenReady(client.url(s"http://localhost:$port/ping/ping").get()) {result ⇒
-          result.status shouldBe 200
+          result.status mustBe 200
         }
       }
     }
 
```

Comment:
We need to review these tests later. But I will accept this change for now.

Reply:
thanks, the other comments have followed